ui/app/src/modules/Content/Dependencies/DependenciesDialogUI.tsx

```diff
@@ -0,0 +1,473 @@
+                    {popupState => (
+                      <React.Fragment>
+                        <IconButton
+                          edge="end"
+                          aria-label="delete"
+                          {...bindTrigger(popupState)}
+                        >
+                          <MoreVertIcon/>
+                        </IconButton>
+                        <Menu {...bindMenu(popupState)}>
+                          {
+                            isEditableItem(dependency.uri) &&
+                            <MenuItem onClick={() => {
+                              handleEditorDisplay(dependency);
+                              popupState.close();
+                            }}>
+                              <FormattedMessage
+                                id="dependenciesDialog.edit"
+                                defaultMessage="Edit"
+                              />
+                            </MenuItem>
+                          }
+                          <MenuItem onClick={() => {
+                            setState({ item: dependency });
+                            popupState.close();
+                          }}>
+                            <FormattedMessage
+                              id="dependenciesDialog.dependencies"
+                              defaultMessage="Dependencies"
+                            />
+                          </MenuItem>
+                          <MenuItem
+                            onClick={popupState.close}>   {/* TODO: pending, waiting for new history dialog */}
+                            <FormattedMessage
+                              id="dependenciesDialog.history"
+                              defaultMessage="History"
+                            />
+                          </MenuItem>
+                        </Menu>
+                      </React.Fragment>
+                    )}
+                  </PopupState>
+                </ListItemSecondaryAction>
+              </ListItem>
+            )
+          })
+      }
+    </List>
+  )
+}
+
+interface DependenciesDialogUIProps {
+  resource: Resource<Item[]>
+  state: any;
+  setState: Function;
+  open: boolean;
+  handleClose: any;
+  isEditableItem: Function;
+  assetsTypes: any;
+  editDialogConfig: any;
+  setEditDialogConfig: Function;
+  handleEditorDisplay: Function;
+}
+
+function DependenciesDialogUI(props: DependenciesDialogUIProps) {
+  const {
+    resource,
+    state,
+    setState,
+    open,
+    handleClose,
+    isEditableItem,
+    assetsTypes,
+    editDialogConfig,
+    setEditDialogConfig,
+    handleEditorDisplay
+  } = props;
+  const classes = dependenciesDialogStyles({});
+  const { formatMessage } = useIntl();
+
+  return (
+    <Dialog
+      onClose={handleClose}
+      open={open}
+      fullWidth={true}
+      maxWidth={'md'}
+      classes={{
+        root: classes.root,
+        paper: classes.dialogPaper
+      }}
+    >
+      {
+        <>
```

Comment:
What are the curls for here? 
What is the fragment for here?

Reply:
Removed, they were being used before for a validation